joern-cli/frontends/kotlin2cpg/build.sbt

```diff
@@ -0,0 +1,30 @@
+name := "kotlin2cpg"
+
+scalaVersion := "2.13.7"
```

Comment:
you can drop a few things here, which are already defined for the entire build in /build.sbt:

scalaVersion
fork
resolvers
scalacOptions
onChangedBuildSource

Reply:
ah, good catch, thanks, will address